src/translations/index.js

```diff
@@ -22,15 +22,11 @@ const TRANSLATIONS = {
         'recovery-words-storing': 'Write these words on a piece of paper and store them at a safe, offline place.',
         'recovery-words-continue-to-words': 'Continue to Recovery Words',
 
-        'create-heading-choose-identicon': 'Choose your avatar',
-        'create-text-select-avatar': 'Select an avatar for your wallet\'s default account from the selection below.',
-        'create-hint-more-accounts': 'You can add more accounts later.',
-        'create-heading-keyfile': 'This is your Key File',
-        'create-text-keyfile-info': 'Your Key File gives you full access to your wallet. '
-                                  + 'You\'ll need it everytime you log in.',
-        'create-hint-keyfile-password': 'To protect your wallet, first protect it with a password.',
-        'create-heading-backup-account': 'Create a backup',
+        'create-heading-choose-identicon': 'Choose an avatar',
+        'create-heading-create-password': 'Create a password',
+        'create-heading-repeat-password': 'Confirm your password',
```

Comment:
Mh, ok, what is our standard for capitalization of headings?

Reply:
a very good question. I assumed we would not capitalize, as none of the headlines had capitalization before except for `Key File` which we capitalize everywhere. How about doing this (or don't, depending on decision) in another PR once wording is finalized?